File: fleet-pay-source-code 1 (1)/fleet-pay-export/flutter/lib/models/bank_account.dart
import 'package:freezed_annotation/freezed_annotation.dart';

import 'bank_account_type.dart';
// --- start import code ---
  
// --- end import code ---

part 'bank_account.g.dart';
part 'bank_account.freezed.dart';

@unfreezed
class BankAccount with _$BankAccount {
  // --- start init code ---
  
  // --- end init code ---

  factory BankAccount({
    required String id,
    required String userId,
    required String accountNumber,
    required String routingNumber,
    required BankAccountType accountType,
  }) = _BankAccount;

  factory BankAccount.fromJson(Map<String, dynamic> json) => _$BankAccountFromJson(json);

  // --- start extension code ---
  
  // --- end extension code ---

}

File: 1-Future/3 fleet-pay-export/app/src/models/bank_account.ts
import { BankAccountType } from './bank_account_type';

export interface BankAccount {
  id: string,
  userId: string,
  accountNumber: string,
  routingNumber: string,
  accountType: BankAccountType,
  // --- start extra_fields code ---
  
  // --- end extra_fields code ---
}
